oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleAction.java

```diff
@@ -18,6 +18,7 @@
 
 import com.google.common.collect.ImmutableMap;
 import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
```

Comment:
unused import?

Reply:
Yeah in earlier configs I was thinking about using it with a field. Not needed any more and will remove